From 1d35d7325ef31b31bea342e692a2ea5578924aa4 Mon Sep 17 00:00:00 2001
From: Madalin Bucur <madalin.bucur@nxp.com>
Date: Wed, 18 Apr 2018 08:58:09 +0300
Subject: [PATCH 197/741] sdk_fman: disable timer code

commit 2411c7ea5263bf8b6f931977031f39eac2673f55 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The (unused) timer code is no longer compatible with the newer
kernel API. Disabling the incompatible code.

Signed-off-by: Madalin Bucur <madalin.bucur@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_linux.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_linux.c b/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_linux.c
index 992757d45c5b..076eccfe06fb 100644
--- a/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_linux.c
+++ b/drivers/net/ethernet/freescale/sdk_fman/src/xx/xx_linux.c
@@ -614,7 +614,7 @@ uint32_t XX_CurrentTime(void)
     return (jiffies*1000)/HZ;
 }
 
-
+#if 0
 t_Handle XX_CreateTimer(void)
 {
     struct timer_list *p_Timer = (struct timer_list *)XX_Malloc(sizeof(struct timer_list));
@@ -691,6 +691,7 @@ int XX_TimerIsActive(t_Handle h_Timer)
 {
   return timer_pending((struct timer_list *)h_Timer);
 }
+#endif
 
 uint32_t XX_Sleep(uint32_t msecs)
 {
-- 
2.17.1

